http/1.9/modules/ROOT/pages/http-connector-config-acb.adoc

```diff
@@ -0,0 +1,252 @@
++
+* *URI Settings*
+** *URL*
++
+URL to send the request to.
++
+** *Path*
++
+Path for where to send the request.
++
+* *Request*
+** *Body*
++
+Body of the request response message.
++
+** *Headers*
++
+Headers to include in the response message.
++
+** *URI Parameters*
++
+URI parameters used to create the request.
++
+** *Query Parameters*
++
+Query parameters to include in the request.
++
+* *Correlations ID*
++
+Select an option for whether to include an outbound correlation ID in the request. 
++
+. Select the *Advanced* tab to configure the following attributes:
++
+* *Output MIME type*
++
+Defines the format of the request.
++
+* *Parameters*
++
+Add parameters to the request. 
++
+* *Output encoding*
++
+Select the encoding to transform the data to.
++
+* *Streaming Strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:connectors::streaming-strategies.adoc[]
++
+* *Connector Overrides*
+** *Follow redirects*
++
+Specifies whether or not to follow redirects.
++
+** *Send body mode*
++
+Select an option to define whether the request contains a body. If you select *Auto*, the behavior is determined by the defined allowed HTTP methods (GET, HEAD, and OPTIONS does not send a body).
++
+** *Request streaming mode*
++
+Select an option to define whether the request is sent using streaming.
++ 
+** *Response timeout*
++
+Maximum time in milliseconds that the request element blocks the execution of the flow waiting for the HTTP response.
++
+* *Response*
+** *Response validator*
++
+Configures default error handling for the request. This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#SuccessStatusCodeValidator[Success Status Code Validator] and xref:http-documentation.adoc#FailureStatusCodeValidator[Failure Status Code Validator].
+* *Output*
+** *Target Variable*
++
+Name of the variable on which the operation's output is placed.
++
+** *Target Value*
++
+An expression that's evaluated against the operation's output. The outcome of that expression is stored in the target variable.
++ 
+* *Error Mappings*
++
+This attribute is not currently enabled. 
++
+* *Connection*
+** *Reconnection strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#reconnect[Reconnect] and xref:http-documentation.adoc#reconnect[Reconnect Forever]. 
+
+== Configuring Attributes for Load Static Resource
+
+The static resource serves static content to use with HTTP. The request path is used to look up the resource. 
+
+To configure the attributes for HTTP *Load static resource* in the Anypoint Code Builder UI:
+
+. Click the *Load static resource* component on the canvas.
++
+The configuration UI opens.
++
+. In the *General* tab, configure the following attributes:
```

Comment:
In a tab or on? I can't find the answer. Perhaps it doesn't matter.

Reply:
I always say in. I remember having this discussion and I don't remember what the conclusion was. I think you are on a page but in a window, on a screen and in the tab. My mind can be changed.